src/nl/rubensten/texifyidea/intentions/LatexLabelDefiningNewCommand.kt

```diff
@@ -0,0 +1,112 @@
+package nl.rubensten.texifyidea.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiFile
+import nl.rubensten.texifyidea.psi.LatexCommands
+import nl.rubensten.texifyidea.settings.LabelingCommandInformation
+import nl.rubensten.texifyidea.settings.TexifySettings
+import nl.rubensten.texifyidea.settings.labeldefiningcommands.EditLabelDefiningCommand
+import nl.rubensten.texifyidea.util.*
+
+open class LatexLabelDefiningNewCommand : TexifyIntentionBase("Add label defining command to list") {
```

Comment:
I'd suggest changing 'list' to 'settings'

```suggestion
open class LatexLabelDefiningNewCommand : TexifyIntentionBase("Add label defining command to settings") {
```